deployment/terraform/cdn.tf

```diff
@@ -106,3 +112,13 @@ resource "aws_cloudfront_distribution" "cdn" {
     Environment = "${var.environment}"
   }
 }
+
+
+## This resource is only available in Terraform AWS Provider 3.41.0 and later
+# resource "aws_cloudfront_function" "oshub_conditional_redirect" {
+#   name    = "cffn${replace(var.project, " ", "")}${var.environment}OSHubConditionalRedirect"
+#   runtime = "cloudfront-js-1.0"
+#   comment = "Redirect non-embedded map traffic to opensupplyhub.org"
+#   publish = true
+#   code    = "${file("cloudfront-functions/conditional_oshub_redirect.js")}"
+# }
```

Comment:
Left behind for future reference on how this could be automatically provisioned if we have a newer version of Terraform and the AWS provider.